packages/flame/lib/src/gestures/events.dart

```diff
@@ -14,7 +14,7 @@ class EventPosition {
   final Offset? _globalPosition;
 
   /// Coordinates of the event relative to the game position/size and transformations
-  late final Vector2 game = _game.projectOffset(_localPosition);
+  late final Vector2 game = _game.unprojectVector(_localPosition.toVector2());
```

Comment:
Does `late` ensure that this is always lazy here?

Reply:
I think that is exactly what it does! but @erickzanardo can confirm as he wrote that bit

Reply:
As far as I know, `toVector` will be executed only when this is accessed for the first time.

But would be nice to confirm, when I originally wrote this, I used the offset to be sure that no new instance would be created.